src/legacy/ui/public/agg_response/tabify/_get_columns.ts

```diff
@@ -17,56 +17,4 @@
  * under the License.
  */
 
-import { groupBy } from 'lodash';
-import { IAggConfig } from '../../agg_types';
-
-export interface AggColumn {
-  aggConfig: IAggConfig;
-  id: string;
-  name: string;
-}
-
-const getColumn = (agg: IAggConfig, i: number): AggColumn => {
-  return {
-    aggConfig: agg,
-    id: `col-${i}-${agg.id}`,
-    name: agg.makeLabel(),
-  };
-};
-
-/**
- * Builds tabify columns.
- *
- * @param {AggConfigs} aggs - the agg configs object to which the aggregation response correlates
- * @param {boolean} minimalColumns - setting to true will only return a column for the last bucket/metric instead of one for each level
- */
-export function tabifyGetColumns(aggs: IAggConfig[], minimalColumns: boolean) {
-  // pick the columns
-  if (minimalColumns) {
-    return aggs.map((agg, i) => getColumn(agg, i));
-  }
-
-  // supposed to be bucket,...metrics,bucket,...metrics
-  const columns: AggColumn[] = [];
-
-  // separate the metrics
-  const grouped = groupBy(aggs, agg => {
-    return agg.type.type;
-  });
-
-  if (!grouped.buckets) {
-    // return just the metrics, in column format
-    return grouped.metrics.map((agg, i) => getColumn(agg, i));
-  }
-
-  let columnIndex = 0;
-  // return the buckets, and after each place all of the metrics
-  grouped.buckets.forEach(agg => {
-    columns.push(getColumn(agg, columnIndex++));
-    grouped.metrics.forEach(metric => {
-      columns.push(getColumn(metric, columnIndex++));
-    });
-  });
-
-  return columns;
-}
+export { tabifyGetColumns } from '../../../../core_plugins/data/public';
```

Comment:
Can we get rid this of this all together and use this import from `src/legacy/ui/public/agg_response/index.js`?
This will not require changes in multiple places.

Same goes for `src/legacy/ui/public/agg_response/tabify/tabify.ts`